plugins/generator-1.16.4/forge-1.16.4/triggers/block_break.java.ftl

```diff
@@ -1,6 +1,6 @@
 @SubscribeEvent public void onBlockBreak(BlockEvent.BreakEvent event) {
 	Entity entity = event.getPlayer();
-	World world = event.getWorld();
+	World world = (World) event.getWorld();

```

Comment:
This will crash when world is not world, you can't just cast like that.

Reply:
How do I change IWorld to World then?

Reply:
You don't even want to change it, MCreator uses IWorld as world, that's why you need to do type checks and casts in the procedures.